web_edit_user_filter/models/ir_filters.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class IrFilters(models.Model):
+    _inherit = 'ir.filters'
+
+    facet = fields.Text()
+
+    @api.model
+    def get_filters(self, model, action_id=None):
+        res = super().get_filters(model, action_id)
```

Comment:
:thinking: I don't understand what this whole file is doing. I feel like all of it could be done via JS, which is where all filters are constructed and displayed. Can't this be done without this backend code?

Otherwise, in any case it would be helpful some comments explaining this code.

Reply:
It can be done with an extra call but this seemed the easiest and fastest solution. I'll add some comments to the method.